doc/conf.py

```diff
@@ -682,6 +682,22 @@ def notebook_modification_function(notebook_content, notebook_filename):
     # imports inside functions
     code_lines.extend(["import matplotlib", "import pandas"])
 
+    # Work around https://github.com/jupyterlite/pyodide-kernel/issues/166
+    # and https://github.com/pyodide/micropip/issues/223 by installing the
+    # dependencies first, and then scikit-learn from Anaconda.org.
+    if "dev" in release:
+        dev_docs_specific_code = [
+            "import micropip",
```

Comment:
I was under the impression that inside JupyterLite `piplite` was preferred to `micropip`, although I don't know what the difference is. Do you have an opinion on this?

Reply:
Yes, that makes sense. The difference is that `piplite` has some extra handling for PyPI as an index and provides a CLI over `micropip`. We still can't use the CLI because of the linked issues, but I can switch to `piplite.install()`

Reply:
Addressed in 6876dea8e39c9e5fec2abadf763f6f3e52b145a4